### PR TITLE
all: fix doc of var ErrChecksum

### PR DESCRIPTION
**Pull request by ideapark**
Signed-off-by: Park Zhou <ideapark@139.com>

### CODE REVIEW

File: errors.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ var (
 	// Deprecated: Use the error variables defined in the bbolt/errors package.
 	ErrVersionMismatch = errors.ErrVersionMismatch
 
-	// ErrChecksum is returned when either meta page checksum does not match.
+	// ErrChecksum is returned when either meta page or checksum does not match.
```
Comment (ahrtr) [inline errors.go:30]: The change isn't correct. However I agree that we should make the description clearer.
```suggestion
	// ErrChecksum is returned when a checksum mismatch occurs on either of the two meta pages.
```

File: errors/errors.go [hunk 2]
```diff
@@ -21,7 +21,7 @@ var (
 	// different version of Bolt.
 	ErrVersionMismatch = errors.New("version mismatch")
 
-	// ErrChecksum is returned when either meta page checksum does not match.
+	// ErrChecksum is returned when either meta page or checksum does not match.
```
Comment (ahrtr) [inline errors/errors.go:24]: ditto

Comment (ideapark) [reply errors/errors.go]: Updated per your comments

**Review by ahrtr — APPROVED** [review]
lgtm

Thanks

**Pull request merged**